app/gsheets_reader/quickstart.py

```diff
@@ -0,0 +1,30 @@
+"""
+Shows basic usage of the Sheets API. Prints values from a Google Spreadsheet.
+"""
+from __future__ import print_function
+from apiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+
+# Setup the Sheets API
+SCOPES = 'https://www.googleapis.com/auth/spreadsheets.readonly'
+store = file.Storage('credentials.json')
+creds = store.get()
+if not creds or creds.invalid:
+    flow = client.flow_from_clientsecrets('client_secret.json', SCOPES)
+    creds = tools.run_flow(flow, store)
+service = build('sheets', 'v4', http=creds.authorize(Http()))
+
+# Call the Sheets API
+SPREADSHEET_ID = '1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms'
+RANGE_NAME = 'Class Data!A2:E'
+result = service.spreadsheets().values().get(spreadsheetId=SPREADSHEET_ID,
+                                             range=RANGE_NAME).execute()
```

Comment:
accidental extra line

Reply:
@sewagodimo , isn't this just to accommodate for the line wrap of `range=RANGE_NAME...`
?